content/intro-to-storybook/react-native/pt/data.md

```diff
@@ -4,10 +4,9 @@ tocTitle: 'Dados'
 description: 'Aprenda como conectar dados ao seu componente de UI'
 ---
 
-Até agora, criamos componentes sem estado isolados – ótimos para o Storybook, mas, em última análise, não são úteis até fornecermos a eles alguns dados em nosso aplicativo.
-
-Este tutorial não tem foco nos detalhes da criação de um aplicativo, então não vamos nos aprofundar nesses detalhes aqui. Mas vamos dedicar um momento para examinar um padrão comum de possibilitar dados em componentes conectados.
+Até agora foram criados componentes sem estado e isolados, o que é fantástico para Storybook, mas em última análise não são úteis até que for fornecido algum tipo de dados da aplicação
 
+Este tutorial não foca particularmente na construção de uma aplicação, como tal não vamos aprofundar muito este aspeto. Mas será feito um aparte para olhar para um padrão comum para ligação de dados com componentes conectados.
 ## Componentes conectados
```

Comment:
@Wanderson77 can you add a line above the Componentes conectados heading so that it can render accurately when this is published?